app/post/modify/post-editor.directive.js

```diff
@@ -62,7 +65,6 @@ function PostEditorController(
     $scope.setVisibleStage = setVisibleStage;
     $scope.fetchAttributes = fetchAttributes;
     $scope.fetchStages = fetchStages;
-    $scope.allowedChangeStatus = allowedChangeStatus;
```

Comment:
Do we not need this in the view anymore? not even to show/hide the publish control?